wire-ios-request-strategy/Sources/Request Strategies/Conversation/Actions/AddParticipantActionHandlerTests.swift

```diff
@@ -314,13 +314,11 @@ class AddParticipantActionHandlerTests: MessagingTestBase {
             action.onResult {
                 switch $0 {
                 case .failure(.nonFederatingDomains([applesDomain, bananasDomain])):
-                    break
+                    isDone.fulfill()
```

Comment:
If we only fulfill the expectation in the expected case, then the test will probably wait around 30seconds before failing in any other case. If we fulfill the expectation on all results, then the test will finish almost immediately but we we still know about the unexpected result.

Reply:
I reverted the changes